packages/zeebe-element-templates-json-schema/src/error-messages.json

```diff
@@ -444,5 +444,42 @@
       "properties"
     ],
     "errorMessage": "\"formId\" and \"externalReference\" cannot be used together"
+  },
+  {
+    "path": [
+      "definitions",
+      "properties",
+      "allOf",
+      1,
+      "items",
+      "allOf",
+      22,
+      "allOf",
+      2,
+      "then",
+      "properties",
+      "value"
+    ],
+    "errorMessage": "Invalid value for priority. Must be between 0 and 100."
```

Comment:
We usually don't place full stop at the end of the error message. Let's keep it consistent.